File: 08_leet_code/daily/isPalindrome.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        xList: List[int] = list(map(int, str(x)))

        for i in range(len(xList)):
            j = len(xList) - 1 - i
            
            if xList[i] != xList[j]:
                return False
        
        return True

File: 08_leet_code/daily/is_palindrome.java
class Solution {
    public boolean isPalindrome(int x) {
        String xStr = Integer.toString(x);
        for (int i = 0; i < xStr.length(); i++) {
            int j = xStr.length() - 1 - i;
            if (xStr.charAt(i) != xStr.charAt(j)) {
                return false;
            }
        }  
        return true;
    }
}